plugin.video.carnet-meduza/resources/lib/carnet-meduza.py

```diff
@@ -0,0 +1,270 @@
+# -*- coding: utf-8 -*-
+import os, sys
+import urllib, urlparse, json
+import xbmcgui, xbmcplugin, xbmcaddon
+import requests
+from register import device
```

Comment:
It looks like you're not using 'device' any longer - it looks like you read/write directly to addon settings. If so, please remove device.py and the memstorage.py, I don't see them being used.